tests/test_context.py

```diff
@@ -18,55 +18,86 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-from __future__ import absolute_import
-
-import pytest
-from tornado import gen
-
-from tchannel import TChannel
-from tchannel import Response
-from tchannel import schemes
-
-
-@pytest.mark.gen_test
-@pytest.mark.call
-def test_context_should_carry_tracing_info():
-    context = [None, None]
-    server = TChannel(name='server')
-
-    @server.register(scheme=schemes.RAW)
-    @gen.coroutine
-    def endpoint1(request):
-        yield server.call(
-            scheme=schemes.RAW,
-            service='server',
-            arg1='endpoint2',
-            arg2='req headers',
-            arg3='req body',
-            hostport=server.hostport,
-        )
-        context[0] = server.context_provider.get_current_context()
-        raise gen.Return(Response('resp body', 'resp headers'))
-
-    @server.register(scheme=schemes.RAW)
-    def endpoint2(request):
-        context[1] = server.context_provider.get_current_context()
-        return Response('resp body', 'resp headers')
-
-    server.listen()
-
-    # Make a call:
-
-    tchannel = TChannel(name='client')
-
-    yield tchannel.call(
-        scheme=schemes.RAW,
-        service='server',
-        arg1='endpoint1',
-        arg2='req headers',
-        arg3='req body',
-        hostport=server.hostport,
-    )
-
-    assert context[0].parent_tracing.name == 'endpoint1'
-    assert context[1].parent_tracing.name == 'endpoint2'
+# from __future__ import absolute_import
+#
+# import threading
```

Comment:
I think it needs to be deleted. The new `context.py` no longer provides the same propagation features as before, its methods are only for API backwards compatibility.